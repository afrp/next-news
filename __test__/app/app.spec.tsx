import AppPage from "@/app/page";
import {render, screen} from '@testing-library/react'
import { describe } from "node:test";

describe('App Page', () =>{
    it('should render', () =>{
        const page = render(<AppPage/>);
        expect(page).toMatchSnapshot();
    })
})