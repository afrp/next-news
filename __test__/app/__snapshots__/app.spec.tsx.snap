// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="main"
      >
        <div
          class="description"
        >
          <div
            class="boxSearch"
          >
            <input
              class="form-controls inputSearch"
              disabled=""
              id="id"
              placeholder="Search News"
              type="text"
              value=""
            />
            <svg
              fill="none"
              height="24"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M14.7106 14.4999H15.2928L19.3862 18.6126L19.3871 18.6134C19.6018 18.8282 19.6018 19.1816 19.3871 19.3963C19.1723 19.6111 18.8189 19.6111 18.6042 19.3963L14.5006 15.2928V14.7099V14.5081L14.3605 14.3628L14.2793 14.2785L14.3636 14.3598L14.5088 14.4999H14.7106ZM14.051 13.4045L13.753 13.7522L13.4052 14.0503C12.1119 15.1589 10.3499 15.7309 8.47385 15.4169C5.91162 14.9836 3.86022 12.8392 3.54698 10.2596L3.54689 10.2589C3.06657 6.35167 6.3524 3.06584 10.2596 3.54616L10.2604 3.54625C12.84 3.85949 14.9844 5.91089 15.4176 8.47311C15.7316 10.3492 15.1596 12.1111 14.051 13.4045ZM4.50063 9.49989C4.50063 12.266 6.73448 14.4999 9.50063 14.4999C12.2668 14.4999 14.5006 12.266 14.5006 9.49989C14.5006 6.73375 12.2668 4.49989 9.50063 4.49989C6.73448 4.49989 4.50063 6.73375 4.50063 9.49989Z"
                fill="currentColor"
                stroke="currentColor"
              />
            </svg>
          </div>
        </div>
        <div
          class="loading"
        >
          Loading...
        </div>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="main"
    >
      <div
        class="description"
      >
        <div
          class="boxSearch"
        >
          <input
            class="form-controls inputSearch"
            disabled=""
            id="id"
            placeholder="Search News"
            type="text"
            value=""
          />
          <svg
            fill="none"
            height="24"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M14.7106 14.4999H15.2928L19.3862 18.6126L19.3871 18.6134C19.6018 18.8282 19.6018 19.1816 19.3871 19.3963C19.1723 19.6111 18.8189 19.6111 18.6042 19.3963L14.5006 15.2928V14.7099V14.5081L14.3605 14.3628L14.2793 14.2785L14.3636 14.3598L14.5088 14.4999H14.7106ZM14.051 13.4045L13.753 13.7522L13.4052 14.0503C12.1119 15.1589 10.3499 15.7309 8.47385 15.4169C5.91162 14.9836 3.86022 12.8392 3.54698 10.2596L3.54689 10.2589C3.06657 6.35167 6.3524 3.06584 10.2596 3.54616L10.2604 3.54625C12.84 3.85949 14.9844 5.91089 15.4176 8.47311C15.7316 10.3492 15.1596 12.1111 14.051 13.4045ZM4.50063 9.49989C4.50063 12.266 6.73448 14.4999 9.50063 14.4999C12.2668 14.4999 14.5006 12.266 14.5006 9.49989C14.5006 6.73375 12.2668 4.49989 9.50063 4.49989C6.73448 4.49989 4.50063 6.73375 4.50063 9.49989Z"
              fill="currentColor"
              stroke="currentColor"
            />
          </svg>
        </div>
      </div>
      <div
        class="loading"
      >
        Loading...
      </div>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
